src/unix/fs.c

```diff
@@ -393,6 +396,28 @@ static ssize_t uv__fs_readlink(uv_fs_t* req) {
   char* buf;
   char* newbuf;
 
+#if defined(UV__FS_PATH_MAX_FALLBACK)
+  /* We may not have a real PATH_MAX.  Read size of link.  */
+  struct stat st;
+  int ret;
+  ret = lstat(req->path, &st);
+  if (ret != 0)
```

Comment:
Don't we need to say why the system call failed, with `errno`?

Reply:
`lstat` will have left `errno` set already to whatever caused it to fail.